satellite/satellitedb/migrate.go

```diff
@@ -1067,6 +1067,36 @@ func (db *satelliteDB) PostgresMigration() *migrate.Migration {
 					ON CONFLICT(project_id, interval_month) DO UPDATE SET egress_allocated = EXCLUDED.egress_allocated::bigint;`,
 				},
 			},
+			{
+				DB:          db.DB,
+				Description: "add separate bandwidth column",
+				Version:     107,
+				Action: migrate.Func(func(ctx context.Context, log *zap.Logger, db tagsql.DB, tx tagsql.Tx) error {
+					// ensure migration is not flattened and executed separate from 108
```

Comment:
can you say something more why it cannot be as one migration step?

Reply:
Cockroach cannot add a column and set the entries in one go.  It have to be seperate steps.
In addition it has to be done in a migrate.Func as our flattenMigration function instead would again condense them into one.